libs/portal/shared/features/form/ui/date-field.tsx

```
@@ -67,7 +67,7 @@ export function DateField({
               </div>
 
               <div className="flex gap-x-3 w-full py-1.5">
-                <img src={calendarIcon} className="h-6 w-6" />
+                <img src={calendarIcon} className="size-6" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)